jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -183,6 +180,19 @@ def _on_glue_y_display_unit_changed(self, y_unit_str):
                 self.flux_unit.selected,
                 self.angle_unit.selected
             )
+            if self.angle_unit.selected == 'pix':
+                mouseover_unit = self.flux_unit.selected
+            else:
+                mouseover_unit = self.sb_unit_selected
```

Comment:
@cshanahan1 - this logic will probably need to change for your effort once images/cubes are guaranteed to always be in SB units